packages/mobile/src/exchange/CeloGoldHistoryChart.tsx

```diff
@@ -0,0 +1,309 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import BigNumber from 'bignumber.js'
+import _ from 'lodash'
+import React, { useCallback, useEffect, useState } from 'react'
+import { WithTranslation } from 'react-i18next'
+import {
+  ActivityIndicator,
+  Dimensions,
+  LayoutChangeEvent,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import { LineChart } from 'react-native-chart-kit'
+import { useDispatch } from 'react-redux'
+import { syncCeloGoldExchangeRateHistory } from 'src/exchange/actions'
+import { Namespaces, withTranslation } from 'src/i18n'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { useDollarsToLocalAmount, useLocalCurrencyCode } from 'src/localCurrency/hooks'
+import useSelector from 'src/redux/useSelector'
+import { getLocalCurrencyDisplayValue } from 'src/utils/formatting'
+
```

Comment:
Nit, spacing here. Can you configure vim to organize imports in the same way as vscode?